make build error: invalid reference format

-e ==> Building sentry-onpremise:latest
Sending build context to Docker daemon  32.26kB
Step 1/2 : ARG IMAGE
Step 2/2 : FROM ${IMAGE}-onbuild
invalid reference format
make: *** [build] Error 1

I have exactly the same issue:

> Building sentry-onpremise:latest
> Sending build context to Docker daemon  32.26kB
> Step 1/2 : ARG IMAGE
> Step 2/2 : FROM ${IMAGE}-onbuild
> invalid reference format
> Makefile:8: recipe for target 'build' failed
> make: *** [build] Error 1

With:

- Ubuntu 18.04.2 LTS
- Docker version 18.09.7, build 2d0083d

And following this https://docs.sentry.io/server/installation/docker/

--- a/README.md
+++ b/README.md
@@ -4,7 +4,7 @@ Official bootstrap for running your own [Sentry](https://sentry.io/) with [Docke
 
 ## Requirements
 
- * Docker 1.10.0+
+ * Docker 17.05.0+
  * Compose 1.17.0+
 
 ## Minimum Hardware Requirements:

--- a/install.sh
+++ b/install.sh
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-MIN_DOCKER_VERSION='1.10.0'
+MIN_DOCKER_VERSION='17.05.0'
 MIN_COMPOSE_VERSION='1.17.0'
 MIN_RAM=3072 # MB
 ENV_FILE='.env'

